### PR TITLE
[ticket/15825] Add core.acp_manage_forums_move_content_sql_before

### PR DESCRIPTION
**Pull request by 3D-I**
PHPBB3-15825

Checklist:

- [x] Correct branch: master for new features; 3.2.x for fixes
- [ ] Tests pass
- [x] Code follows coding guidelines: [master](https://area51.phpbb.com/docs/dev/master/development/coding_guidelines.html) and [3.2.x](https://area51.phpbb.com/docs/dev/3.2.x/development/coding_guidelines.html)
- [x] Commit follows commit message [format](https://area51.phpbb.com/docs/dev/3.2.x/development/git.html)

Tracker ticket (set the ticket ID to **your ticket ID**):

https://tracker.phpbb.com/browse/PHPBB3-15825

### CODE REVIEW

**Comment by 3D-I** [comment]
Not sure about the failing Travis-ci check, someone pls?

**Comment by dark-1** [comment]
> Not sure about the failing Travis-ci check, someone pls?

Here : https://github.com/3D-I/phpbb/pull/1     
    
Best regards 👍

File: phpBB/includes/acp/acp_forums.php [hunk 1]
```diff
@@ -1541,6 +1541,16 @@ function move_forum_content($from_id, $to_id, $sync = true)
...
 		$table_ary = array(LOG_TABLE, POSTS_TABLE, TOPICS_TABLE, DRAFTS_TABLE, TOPICS_TRACK_TABLE);
 
+		/**
+		 * Perform additional actions before move forum content
+		 *
+		 * @event core.acp_manage_forums_move_content_sql_before
+		 * @var	array	table_ary	Array of tables from which forum_id will be updated
+		 * @since 3.2.4-RC1
+		 */
+		$vars = array('table_ary',);
```
[the remark this answers is not in the document]
Comment (dark-1) [reply phpBB/includes/acp/acp_forums.php]: > You have a stray comma here.

Yea! I made a PR for the same.
Here:


> ...
> Here : [3D-I#1](https://github.com/3D-I/phpbb/pull/1)
> ...

Best regards 👍

**Comment by 3D-I** [comment]
Thx all.

**Comment by dark-1** [comment]
> Thx all.

Welcome 😄    
But make a PR with Heading `[ticket/15825] Add core.acp_manage_forums_move_content_sql_before`   
to get accepted by CI.
Look at this Line : https://travis-ci.org/phpbb/phpbb/jobs/436180458#L695   

It's Funny though... due to wrong heading the whole PR is rejected by CI. 😆 

Best regards 👍

**Comment by DavidIQ** [comment]
@dark-1 not really funny at all. It's in the [git contribution guidelines](https://area51.phpbb.com/docs/dev/3.2.x/development/git.html), which is linked to in every new pr, and there is a commit hook that one can set up to avoid these sorts of errors. That the build verifies this is pretty much standard for large projects that actually want the guidelines to be followed.

@3D-I you'll need to do this to change the commit message:
```
git commit --amend #your default text editor will open where you can change the commit message#
git commit push --force
```

**Comment by dark-1** [comment]
> @dark-1 not really funny at all. It's in the [git contribution guidelines](https://area51.phpbb.com/docs/dev/3.2.x/development/git.html), which is linked to in every new pr, and there is a commit hook that one can set up to avoid these sorts of errors. That the build verifies this is pretty much standard for large projects that actually want the guidelines to be followed.


Sir,
Do you mean this : `commit-msg hook` ???
Thanks for tell me.
BTW I wasn't trying to be rude to anyone though, my Apologies.

Best regards 👍

**Comment by 3D-I** [comment]
> 
> 
> @3D-I you'll need to do this to change the commit message:
> 
> ```
> git commit --amend #your default text editor will open where you can change the commit message#
> git commit push --force
> ```
Thx David, did it via TortoiseGit http://prntscr.com/l1bfp2 I hope it's ok now. :+1:

**Comment by dark-1** [comment]
Some thing wrong again for last commit : https://github.com/phpbb/phpbb/pull/5398/commits/75a801fa71b95f3e0680e907e60df99f04bf5352

Error Message : `Unexpected EOF encountered`
at : https://travis-ci.org/phpbb/phpbb/jobs/436196660#L693

Best regards 👍

**Pull request closed without merging**